tests/vyper_setup.sol

```diff
@@ -0,0 +1,15 @@
+      function write(bytes memory _code) public returns (address target) {
+        assembly {
+            target := create(0, add(_code, 0x20), mload(_code))
+        }
+    }
```

Comment:
Nitpick: can you make indentation consistent across the files? Not important though.